ui/app/src/state/reducers/dialogs/delete.ts

```diff
@@ -47,7 +49,7 @@ export default createReducer<GlobalState['dialogs']['delete']>(initialState, {
     ...state,
     ...payload
   }),
-  [closeDeleteDialog.type]: (state) => ({ ...state, open: false, disableQuickDismiss: false }),
+  [closeDeleteDialog.type]: (state) => ({ ...state, open: false, isSubmitting: false }),
```

Comment:
Shouldn't we be resetting them all? This goes for the other dialog reducers or containers too.